module2/lessons/forms.md

```diff
@@ -234,110 +252,144 @@ class ArtistsController < ApplicationController
+
+**app/controllers/artists_controller.rb**
 
 ```ruby
 class ArtistsController < ApplicationController
+  def index
+  end
+  
   def new
   end
 
   def create
     Artist.create(name: params[:name])
-    redirect_to '/artists'
+    redirect_to "/artists"
   end
 end
 ```
 
 And now our test should be passing!
 
-Let's do a little bit of refactoring before we call this feature complete. As an experiment try to replace the line in the controller `Artist.create(name: params[:name])` with `Artist.create(params)`. We should get a `ForbiddenAttributesError`. Take a closer look at that `params` object, using a `binding.pry` - at the very end, you will see `permitted: false`.  This means that we cannot use the params 'hash' directly to create or update records in our database. Rails is trying to protect us from malicious users by not allowing us to drop our all of our params directly into a new object that will be saved into our database, so we need to be explicit about which params we are accepting. The syntax `Artist.create(name: params[:name])` will do that, but as our objects get more complex this syntax will get very verbose. Imagine if an artist also had a hometown, years active, genre, etc. The syntax would look like:
+Let's do a little bit of refactoring before we call this feature complete. As an experiment try to replace the line in the controller `Artist.create(name: params[:name])`  with `Artist.create(params)`. We should get a `ForbiddenAttributesError`. Take a closer look at that `params` object, using a `binding.pry` - at the very end, you will see `permitted: false`. This means that we cannot use the params 'hash' directly to create or update records in our database. Rails is trying to protect us from malicious users by not allowing us to drop our all of our params directly into a new object that will be saved into our database, so we need to be explicit about which params we are accepting. The syntax `Artist.create(name: params[:name])` will do that, but as our objects get more complex this syntax will get very verbose. Imagine if an artist also had a hometown, years active, genre, etc. The syntax would look like:
 
 ```ruby
 Artist.create(name: params[:name], hometown: params[:hometown], years_active: params[:years_active], genre: params[:genre])
 ```
 
-Rather than this long syntax, rails gives us a much nicer way to do this called **strong params**:
+Rather than this long syntax, rails gives us a much nicer way to do this called **strong params.**
+
+**app/controllers/artists_controller.rb**
 
 ```ruby
 class ArtistsController < ApplicationController
+  def index
+  end
+  
   def new
   end
 
   def create
-    Artist.create(artist_params)
-    redirect_to '/artists'
+    Artist.create(name: params[:name])
```

Comment:
I might just be really tired, but shouldn't this still be `artist_params`? This section appears to be about teaching strong params but I don't see where we are using the `artist_params` method anywhere with this change. If so, then I think this also needs to be changed on line 322?